cmd/drift.go

```diff
@@ -30,6 +32,9 @@ var (
 				Profile:    driftProfile,
 				OutputPath: driftOutputPath,
 			})
+			if err := analytics.Capture("drift", c.Providers, nil, diag.FromError(err, diag.INTERNAL)); err != nil {
+				log.Warn().Err(err).Msg("analytics send failed")
```

Comment:
I don't think analytics send failed messages should be in the log unless we specify `--debug-telemetry`. So the logger should be inside the analytics package and this shouldn't return an `error`.

Reply:
fixed